content/en/docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -100,32 +89,17 @@ process will result in kubelet crashlooping with the following error:
 could not restore state from checkpoint: configured policy "static" differs from state checkpoint policy "none", please drain this node and delete the CPU manager checkpoint file "/var/lib/kubelet/cpu_manager_state" before restarting Kubelet
 ```
 
-### None policy
-
-The `none` policy explicitly enables the existing default CPU
-affinity scheme, providing no affinity beyond what the OS scheduler does
-automatically.  Limits on CPU usage for
-[Guaranteed pods](/docs/tasks/configure-pod-container/quality-service-pod/) and
-[Burstable pods](/docs/tasks/configure-pod-container/quality-service-pod/)
-are enforced using CFS quota.
-
-### Static policy
-
-The `static` policy allows containers in `Guaranteed` pods with integer CPU
-`requests` access to exclusive CPUs on the node. This exclusivity is enforced
-using the [cpuset cgroup controller](https://www.kernel.org/doc/Documentation/cgroup-v1/cpusets.txt).
-
 {{< note >}}
-System services such as the container runtime and the kubelet itself can continue to run on these exclusive CPUs.  The exclusivity only extends to other pods.
-{{< /note >}}
-
-{{< note >}}
-CPU Manager doesn't support offlining and onlining of
-CPUs at runtime. Also, if the set of online CPUs changes on the node,
-the node must be drained and CPU manager manually reset by deleting the
+if the set of online CPUs changes on the node, the node must be drained and CPU manager manually reset by deleting the
 state file `cpu_manager_state` in the kubelet root directory.
 {{< /note >}}
 
+#### None policy configuration
```

Comment:
```suggestion
#### `none` policy configuration
```
?

Reply:
Not sure if this is right.

Reply:
I think it is, because `none` is the name of the policy. Applied.